aiven/client/client.py

```diff
@@ -174,11 +179,28 @@ def verify(
                 )
                 time.sleep(0.2)
 
+        return self._process_response(response=response, op=op, path=path, result_key=result_key)
+
+    @staticmethod
+    def build_path(*parts: str) -> str:
+        return "/" + "/".join(quote(part, safe="") for part in parts)
```

Comment:
I don't think it worth it here. I've always considered `os.path.join()` as an incapsulation of building path for different OSes: choosing `/` vs `\`, setting correct root like `C:` vs `/`, etc. Also since this util is supposed to be used on Windows we may face some bugs related to it.

Reply:
I agree here, `os.path.join()` is specifically for concatenating file paths using operator system specific conventions. HTTP request URLs are a different thing, and the ways these two are built don't necessarily match.